Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1649,7 +1650,10 @@ static void SetMetadataFeatureTableParameterValues(
        encodedFeatureTable.encodedProperties) {
 
     pMaterial->SetTextureParameterValueByInfo(
-        FMaterialParameterInfo(FName(encodedProperty.name), association, index),
+        FMaterialParameterInfo(
+            FName(createHlslSafeName(encodedProperty.name)),
+            association,
+            index),
```

Comment:
This is how the parameter name is created for encoded property array textures. As a reminder, encodedProperty.name is built as follows: "FTB_(feature table name)_(property name)". 

Two points about the name sanitation:
- Do parameter names need to be sanitized? I haven't seen material parameters not work due to the name (hlsl identifiers on the other hand definitely should be consistently sanitized), but it couldn't hurt to sanitize it anyways.  
- If we are going to sanitize them, we should do the same in `SetMetadataParameterValues` below as well.